tests/integration/test_provider.py

```diff
@@ -243,7 +243,7 @@ async def test_connection_updated_on_tls_enabled(ops_test: OpsTest, app_charm: P
 
     # deploying tls
     tls_config = {"ca-common-name": "kafka"}
-    await ops_test.model.deploy(TLS_NAME, channel="edge", config=tls_config)
+    await ops_test.model.deploy(TLS_NAME, channel="edge", config=tls_config, revision=163)
```

Comment:
**question:** What happened here?

Reply:
Ah, yes, I explained it in the other PR but not here. I apologize for that.

We have a message in the internal Debug MM channel about this issue. The latest revisions make the charm hanging on waiting for the certificates, and the test times out